README.md

```diff
@@ -73,9 +73,42 @@ development.
 - [IBC Go implementation][ibc-go]
 - [Protobuf definitions in Rust][ibc-proto-rs]
 
+## `ibc-rs` in the wild
+
+The IBC protocol is integrated in numerous networks and projects. The `ibc-rs` implementation specifically
+is adopted for IBC integration, testing, or development in the following use-cases.
+
+|  # | Integration use-Case   |     Team      | Used-Modules |                 Links                 |          Phase           |
+|---:|------------------------|:-------------:|--------------|:-------------------------------------:|:------------------------:|
+|  1 | Solana                 | ComposableFin | all          |    [Picasso Network][pica-network]    |    :white_check_mark:    |
```

Comment:
Could you please rename the Team from ComposableFin to either: `Composable` or `Composable Foundation`

Reply:
done